src/modules/person/api/person.controller.ts

```diff
@@ -52,28 +52,74 @@ import { PersonendatensatzResponse } from './personendatensatz.response.js';
 @ApiBearerAuth()
 @ApiOAuth2(['openid'])
 @Controller({ path: 'personen' })
 export class PersonController {
+    public readonly ROOT_ORGANISATION_ID: string;
+
     public constructor(
         private readonly personenkontextUc: PersonenkontextUc,
         private readonly personRepository: PersonRepository,
         private readonly personFactory: PersonFactory,
+        private readonly personUc: PersonUc,
         @Inject(getMapperToken()) private readonly mapper: Mapper,
-    ) {}
+        config: ConfigService<ServerConfig>,
+    ) {
+        this.ROOT_ORGANISATION_ID = config.getOrThrow<DataConfig>('DATA').ROOT_ORGANISATION_ID;
+    }
+
+    //check that logged-in user is allowed to update person
+    private async getPersonOrThrowException(
+        params: PersonByIdParams,
+        permissions: PersonPermissions,
+    ): Promise<Person<true>> {
+        const getPersonIfAllowed: Result<Person<true>> = await this.personUc.getPersonIfAllowed(
+            params.personId,
+            permissions,
+        );
+        if (!getPersonIfAllowed.ok) {
+            throw SchulConnexErrorMapper.mapSchulConnexErrorToHttpException(
+                SchulConnexErrorMapper.mapDomainErrorToSchulConnexError(
+                    new EntityNotFoundError('Person', params.personId),
+                ),
+            );
+        }
+        return getPersonIfAllowed.value;
+    }
 
     @Post()
     @HttpCode(HttpStatus.CREATED)
     @ApiCreatedResponse({ description: 'The person was successfully created.', type: PersonendatensatzResponse })
-    @ApiBadRequestResponse({ description: 'A username was given. Creation with username is not supported' })
+    @ApiBadRequestResponse({ description: 'A username was given. Creation with username is not supported.' })
     @ApiUnauthorizedResponse({ description: 'Not authorized to create the person.' })
     @ApiForbiddenResponse({ description: 'Insufficient permissions to create the person.' })
+    @ApiNotFoundResponse({ description: 'Insufficient permissions to create the person.' })
     @ApiInternalServerErrorResponse({ description: 'Internal server error while creating the person.' })
-    public async createPerson(@Body() params: CreatePersonBodyParams): Promise<PersonendatensatzResponse> {
+    public async createPerson(
+        @Body() params: CreatePersonBodyParams,
+        @Permissions() permissions: PersonPermissions,
+    ): Promise<PersonendatensatzResponse> {
+        // Find all organisations where user has permission
+        const organisationIDs: OrganisationID[] | undefined = await permissions.getOrgIdsWithSystemrecht(
+            [RollenSystemRecht.PERSONEN_VERWALTEN],
+            true,
+        );
+        if (!organisationIDs || organisationIDs.length < 1) {
+            throw SchulConnexErrorMapper.mapSchulConnexErrorToHttpException(
+                SchulConnexErrorMapper.mapDomainErrorToSchulConnexError(new EntityNotFoundError('Person')),
+            );
+        }
```

Comment:
`getOrgIdsWithSystemrecht` will never return `undefined`, can remove the extraneous checks here